packages/core/src/components/panel/panel.scss

```diff
@@ -0,0 +1,65 @@
+@use "~dso-toolkit/src/utilities";
+@use "~dso-toolkit/src/variables/units";
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/di";
+
+:host {
+  display: block;
+
+  border: 1px solid colors.$bosgroen-40;
+  border-radius: units.$u1 * 0.5;
+}
+
+@include utilities.box-sizing();
+
+.panel-heading {
+  display: flex;
+  gap: units.$u2;
+  justify-content: space-between;
+  align-items: flex-start;
+
+  padding-block: units.$u1;
+  padding-inline: units.$u2 units.$u1;
+
+  border-block-end: 1px solid colors.$bosgroen-40;
+
+  border-start-start-radius: units.$u1 * 0.5;
+  border-start-end-radius: units.$u1 * 0.5;
+
+  background-color: colors.$bosgroen-10;
+
+  color: colors.$bosgroen;
+
+  .sr-only {
+    @include utilities.sr-only();
+  }
+}
+
+::slotted([slot="heading"]) {
```

Comment:
Ik heb me suf gestaard op wat ik hier anders doe dan het voorbeeld op mdn (https://developer.mozilla.org/en-US/docs/Web/CSS/::slotted), maar ik zie het niet. Deze selector doet iig niets. @tfrijsewijk Wil jij even kijken?

Reply:
Ok! Ga ik doen

Reply:
Ik heb het gevonden. `::slotted()` heeft een speciale specificity. Ondanks dat het selector een hogere specificity heeft wordt de styling overruled.

![image](https://github.com/user-attachments/assets/5caef3fc-fbd3-4ad9-9d96-6f1972194b36)

De h2 en `:slotted()` selectors hebben respectievelijk een specificity van `0,0,1` en `0,2,0`. Heel raar dus.

![image](https://github.com/user-attachments/assets/ecc37e00-d871-47d4-819d-60151195794c)

![image](https://github.com/user-attachments/assets/01052225-0ce1-4a9a-b9b7-a61849a20069)

In het w3c standards github project heb ik hier een issue over gevonden:

https://github.com/w3c/csswg-drafts/issues/6466

Als we deze weg op willen gaan moeten we met `!important` werken. Ik denk dat we dit moeten doen. We zijn dan van onze semi-globale styling problemen af en het component wordt daarmee zelfstandiger. Ik denk dat het gebruik van `!important` een kleine prijs is.

cc @hansgrimm

Reply:
Done (denk ik)

Reply:
Let's check it out...

Reply:
Looking sharp